slices/fontconfig.yaml

```diff
@@ -0,0 +1,26 @@
+package: fontconfig
+
+essential:
+  - fontconfig_copyright
+
+slices:
+  bins:
+    essential:
+      - fontconfig-config_config
+      - libc6_libs
+      - libfontconfig1_libs
+      - libfreetype6_libs
+    contents:
+      /usr/bin/fc-cache:
+      /usr/bin/fc-cat:
+      /usr/bin/fc-conflist:
+      /usr/bin/fc-list:
+      /usr/bin/fc-match:
+      /usr/bin/fc-pattern:
+      /usr/bin/fc-query:
+      /usr/bin/fc-scan:
+      /usr/bin/fc-validate:
```

Comment:
Note: the `postinst` script creates the `/var/cache/fontconfig` directory and (re-)generates the cache files using `fc-cache`.

```sh
if [ "$1" = configure ]; then                                                                                           
  # Force regeneration of all fontconfig cache files.
  mkdir -p /var/cache/fontconfig
  printf "Regenerating fonts cache... "
  fc-cache -s -f -v 1>/var/log/fontconfig.log 2>&1 || (printf "failed.\nSee /var/log/fontconfig.log for more            information.\n"; exit 1)
  printf "done.\n"
fi
```

Since we cannot run binaries through mutation scripts in Chisel, I understand there's no value in creating an empty `/var/cache/fontconfig` directory. But let's please leave a comment in the `contents` here hinting at this behaviour in the `postinst` script.

Reply:
Good catch, thanks. I actually added the empty `/var/cache/fontconfig` directory in a `config` slice as the `fc-cache` command errors out if that directory does  not exist.